src/Microsoft.AspNetCore.Authentication/RemoteAuthenticationHandler.cs

```diff
@@ -203,7 +203,7 @@ protected virtual void GenerateCorrelationId(AuthenticationProperties properties
             var cookieOptions = new CookieOptions
             {
                 HttpOnly = true,
-                SameSite = SameSiteMode.Lax,
+                SameSite = SameSiteMode.Strict,
```

Comment:
> You're right, Strict isn't going to work.

@Tratcher I'm not sure `Lax` will work with `response_mode=form_post` since POST is not considered as a "safe" method by the specification.

@JunTaoLuo did `Lax` work for you?